CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+## Version 2.12.0-beta2
+- [Enable Metric DimensionCapping API for Internal use with standard metric aggregation.]https://github.com/microsoft/ApplicationInsights-dotnet/issues/1244)
+
+## Version 2.12.0-beta1
+- BASE: [New: TelemetryConfiguration now supports Connection Strings]https://github.com/microsoft/ApplicationInsights-dotnet/issues/1221)
+- WEB: [Enhancement to how QuickPulseTelemetryModule shares its ServiceEndpoint with QuickPulseTelemetryProcessor.](https://github.com/microsoft/ApplicationInsights-dotnet-server/pull/1266)
+- WEB: [QuickPulse will support SDK Connection String](https://github.com/microsoft/ApplicationInsights-dotnet/issues/1221)
+- WEB: [Add support for storing EventCounter Metadata as properties of MetricTelemetry](https://github.com/microsoft/ApplicationInsights-dotnet-server/issues/1287)
+- WEB: [New RoleName initializer for Azure Web App to accurately populate RoleName.](https://github.com/microsoft/ApplicationInsights-dotnet-server/issues/1207)
+- NETCORE: Skipping version numbers to keep in sync with Base SDK.
+- NETCORE: [Fix Null/Empty Ikey from ApplicationInsightsServiceOptions overrding one from appsettings.json](https://github.com/microsoft/ApplicationInsights-aspnetcore/issues/989)
+- NETCORE: [Provide ApplicationInsightsServiceOptions for easy disabling of any default TelemetryModules](https://github.com/microsoft/ApplicationInsights-aspnetcore/issues/988)
+- NETCORE: [Added support for SDK Connection String](https://github.com/microsoft/ApplicationInsights-dotnet/issues/1221)
+- NETCORE: [New RoleName initializer for Azure Web App to accurately populate RoleName.](https://github.com/microsoft/ApplicationInsights-dotnet-server/issues/1207)
+- NETCORE: Update to Base/Web/Logging SDK to 2.12.0-beta1
```

Comment:
for ease of navigation - can you link from this page to the old changelog docs?

Reply:
also a note here saying that this is now the combined changelog, and old changelog prior to 2.12 can be found here, here, here and here. (with links)

Reply:
I was about to ask you for your opinion. :)
Would you prefer all the changelogs merged into a single file, or do you think it's good enough to link to them from the new file?